### DISCUSSION TITLE
Github actions runner issue

### DISCUSSION THREAD

**Discussion by MaksadAnnamuradov** [lowlighter/metrics#930] [Q&A]
Hello. So I have tried to run on my self-hosted github actions runner, but I am getting this error on the image:
![image](https://user-images.githubusercontent.com/46725876/156903863-09f88ef4-fb47-46a5-b864-00ec32c3ef87.png)
![image](https://user-images.githubusercontent.com/46725876/156903876-2cf479f5-2eb7-4e9c-97a1-de3493d48f65.png)
![image](https://user-images.githubusercontent.com/46725876/156903887-f2b7a36b-3817-4449-a993-5bf0b6ba377d.png)
And I have set up the METRICS_TOKEN. I am not sure what the error is. Could you help me with it?

**Comment by lowlighter** [comment]
Hi 👋 !

It's kind of hard to debug these errors because they happen before the initial setup 😕 

I'll try to test to reproduce on a self-hosted runner later when I have some free 👍 

(will be tracked in #937)

**Reply by lowlighter** [reply]
It may have been fixed on more recent versions, I didn't encounter this kind of error on v3.22+

However, I did find some stuff that needs to be done on self-runner in order to make the action works:
- (follow steps to add self-hosted runner to repository)
- install docker 
  - ensure working user can use docker correctly (had some trouble with /var/run/docker) 
- install jq 

I documented these steps [here](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/selfhosted.md)
